RetailCoder.VBE/UI/Command/MenuItems/FindSymbolCommandMenuItem.cs

```diff
@@ -22,8 +22,7 @@ public FindSymbolCommandMenuItem(ICommand command)
 
         public override bool EvaluateCanExecute(RubberduckParserState state)
         {
-            return state.Status == ParserState.Ready ||
-                   state.Status == ParserState.Resolving;
+            return state.Status >= ParserState.ResolvedDeclarations;
```

Comment:
`>= ParserState.ResolvedDeclarations` includes error states. I doubt this is intentional.